portablemc/cli/__init__.py

```diff
@@ -37,6 +37,13 @@
 from portablemc.fabric import FabricVersion, FabricResolveEvent
 from portablemc.forge import ForgeVersion, ForgeResolveEvent, ForgePostProcessingEvent, \
     ForgePostProcessedEvent, ForgeInstallError, _NeoForgeVersion
+from portablemc.optifine import (get_offline_versions as optifine_get_versions_offline,
+                                get_compatible_versions as optifine_get_versions,
```

Comment:
Why renaming the two differently, like `get_offline_versions` is reverted `get_versions_offline` in the alias? And for the compatible versions, why is the word "compatible" gone? I know I'm annoying sorry I'm a maniac lol